frontend/src/screens/wallet/WithdrawOnchainFunds.tsx

```diff
@@ -31,14 +38,29 @@ import { request } from "src/utils/request";
 export default function WithdrawOnchainFunds() {
   const [isLoading, setLoading] = React.useState(false);
   const { toast } = useToast();
+  const { data: info } = useInfo();
   const { data: balances } = useBalances();
+  const { data: recommendedFees } = useMempoolApi<{
+    fastestFee: number;
+    halfHourFee: number;
+    economyFee: number;
+    minimumFee: number;
+  }>("/v1/fees/recommended");
   const { data: channels } = useChannels();
   const [onchainAddress, setOnchainAddress] = React.useState("");
   const [amount, setAmount] = React.useState("");
+  const [feeRate, setFeeRate] = React.useState("");
   const [sendAll, setSendAll] = React.useState(false);
+  const [showAdvanced, setShowAdvanced] = React.useState(false);
   const [transactionId, setTransactionId] = React.useState("");
   const [confirmDialogOpen, setConfirmDialogOpen] = React.useState(false);
 
+  React.useEffect(() => {
```

Comment:
I don't think we should do it this way. This will break if the fee rates cannot be loaded.